app/views/bars/show.html.haml

```diff
@@ -1,28 +1,42 @@
-%h1{:align => "center"} #{@bar.name}
+.container
+  .row
+    %h1{:align => "center", class: "full_width"} #{@bar.name}
 
-.row
-  .col-xs-8{style: "margin:0 auto;"}
-    .card
-      =image_tag "chinczyk.png"
-      %br
-      Rating: #{@bar.rating}
-      %br
-      Address: #{@bar.address}
-      %br
-      #{@bar.description}
-      %br
 
-      %small
-        created by: tutaj dodac usera po polaczeniu modeli
-        =time_ago_in_words(@bar.created_at)
-        ago   ||
-        Last updated:
-        =time_ago_in_words(@bar.updated_at)
-        ago
-      %div
-        =link_to "Edit", edit_bar_path(@bar), class: "btn btn-success"
-        =link_to "Delete", bar_path(@bar), method: :delete,
-                    data: {confirm: "Are you sure you want to delete this bar?"},
-                    class: "btn btn-danger"
-      = render "image/new", bar: @bar
+  .row
+    .col-md-6
+      .card.card-show
+        .card-img-top
+          %div{:align => "center"}
+            =image_tag("chinczyk.png", size:'300x300')
+        .card-body
+          %h4.card-title
+            #{@bar.name}
+          %p.card-text
+            Rating:
+            #{@bar.rating}
+          %p.card-text
+            Address:
+            #{@bar.address}
+          %p.card-text
+            #{@bar.description}
+          %small
+            created by:
+            #{@bar.user.username if @bar.user}
+            =time_ago_in_words(@bar.created_at)
+            ago   ||
+            Last updated:
+            =time_ago_in_words(@bar.updated_at)
+            ago
+          %div
+            =link_to "Edit", edit_bar_path(@bar), class: "btn btn-success"
+            =link_to "Delete", bar_path(@bar), method: :delete,
+                        data: {confirm: "Are you sure you want to delete this bar?"},
+                        class: "btn btn-danger"
+    .col-md6
+      %p tutaj maja  isc komentarze
```

Comment:
typically it is //TODO: comment 😉

Reply:
☝️ it is easier to find all stuff that needed to be modified if it's marked as `TODO: ...`